doc/guides/01-getting-started.md

```diff
@@ -0,0 +1,178 @@
+
+While the `web-component loader` will only inject required web components for the styles you need to specify which elements you want to load:
+
+```html
+<head>
+  <!-- Always Include core first -->
+  <link rel="stylesheet" href="https://libs.cartocdn.com/airship-styles/dist/core/core.css">
+  <!-- Include the components used by your app -->
+  <link rel="stylesheet" href="https://libs.cartocdn.com/airship-styles/dist/button/button.css">
+  <link rel="stylesheet" href="https://libs.cartocdn.com/airship-styles/dist/table/table.css">
+  <link rel="stylesheet" href="https://libs.cartocdn.com/airship-styles/dist/tabs/tabs.css">
+  <!-- (optional) Include utils at the end -->
+  <link rel="stylesheet" href="https://libs.cartocdn.com/airship-styles/dist/utilities/utilities.css">
+<head>
+```
+
+
+## Usage from NPM
+
+### Styles and fonts
+
+Just install our packages and use it as you wish:
+
+    npm i @carto/airship-style @carto/airship-icons
+
+Take a look at [sass-loader](https://github.com/webpack-contrib/sass-loader) if you are using webpack. We provide both `dist` and `src` files so you can create a custom [sass](https://sass-lang.com/) build.
+
+### Web components
+
+    npm i @carto/airship-components
+
+You need to call the `defineCustomElements` function to use the web components from npm.
+
+
+```js
+import { defineCustomElements } from '@carto/airship-components';
+
+defineCustomElements(window);
+```
+
+# Using Airship
+Using airship once included in your webpage is very straightforward. 
+
+## Airship Styles
+
+To use airship styles you only need to add CSS classes to the desired HTML fragment.
+
+For example a button can become a Airhip button adding the `as-btn` class.
+
+```html
+<button class="as-btn"> Button </button>
+```
+
+Airship classes follow [BEM](http://getbem.com/introduction/) conventions, this means that the classes can be of the following types:
+
+- **Block**: Standalone entity that is meaningful on its own. (`.block`)
+- **Element**: A part of a block that has no standalone meaning and is semantically tied to its block. (`.block__element`)
+- **Modifier**: Modifies the block appearance or behavior. (`.block--modifier`)
+
+For example to create tabs we use the following classes:
+
+`.as-toolbar-tabs` and `.as-tabs` to generate a **tab** block. `.as-tabs__item` to indicate the items and `.as-tabs__item--active` to highlight the active item.
+
+```html
+ <div class="as-toolbar-tabs as-tabs" role="tablist">
+    <button role="tab" class="as-tabs__item as-tabs__item--active">MAP</button>
+    <button role="tab" class="as-tabs__item">LEFT</button>
+    <button role="tab" class="as-tabs__item">RIGHT</button>
+    <button role="tab" class="as-tabs__item">PANELS</button>
+  </div>
+```
+
+## Airship Icons
+You can use airship icons in two different ways.
+
+### As web font
+
+To display an icon only a class and an `i` tag is required.
```

Comment:
I'd just put the example here

Reply:
I mean that the text is redundant with the example below, or I would change the wording at least.